stripe/util.py

```diff
@@ -232,6 +231,15 @@ def convert_to_stripe_object(resp, api_key=None, stripe_version=None,
         load_object_classes()
     types = OBJECT_CLASSES.copy()
 
+    # If we get a StripeResponse, we'll want to return a
+    # StripeObject with the last_response field filled out with
+    # the raw API response information
+    stripe_response = None
+
+    if isinstance(resp, stripe.stripe_response.StripeResponse):
```

Comment:
@ob-stripe I'm able to reduce the amount of code change needed from changing `APIRequestor.request` by doing this little awkward bit. Essentially unpacking the response body dict inside `convert_to_stripe_object`.

`APIRequestor.request`'s only real consumer is `convert_to_stripe_object` we only need to change it here.

wdyt?

Reply:
👍  Yeah, that sounds reasonable and less annoying than having to change all the call sites. It also maintains the existing behavior of `convert_to_stripe_object` which is nice even if it's an internal function.